rfc8446.xml

```diff
@@ -4073,11 +4076,11 @@ Application Data keys or upon a key update).</t>
 
 <t>For finite field groups, a conventional Diffie-Hellman
 <xref target="DH76"/> computation is performed.
-The negotiated key (Z) is converted to a byte string by encoding in big-endian and
+The negotiated key (Z) is converted to a byte string by encoding in big-endian form and
 left-padded with zeros up to the size of the prime. This byte string is used as the
 shared secret in the key schedule as specified above.</t>
 
-<t>Note that this construction differs from previous versions of TLS which remove
+<t>Note that this construction differs from previous versions of TLS which removed
```

Comment:
Don't they still remove.

Reply:
They do, but they also "don't exist anymore" (sort-of).

Reply:
I would leave it as-is

Reply:
Seems in at least one other place we use past tense, so keeping your change.